backend/algorithms/objects/user.py

```diff
@@ -21,6 +21,7 @@ def __init__(self, data = None):
         self.program: str = None
         self.specialisations: dict[str, int] = {}
         self.year: int = 0
+        self.core_courses: list[str] = []
```

Comment:
Is there currently a way to populate this?

Reply:
no that'll be in the next stage where we add this functionality to the frontend

Reply:
this pr is just outlining the structure etc

Reply:
Ah fair